pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/tablestate/TableStateUtils.java

```diff
@@ -47,6 +81,10 @@ private TableStateUtils() {
    * @return true if all segments for the given table are succesfully loaded. False otherwise
    */
   public static boolean isAllSegmentsLoaded(HelixManager helixManager, String tableNameWithType) {
+    HelixDataAccessor dataAccessor = helixManager.getHelixDataAccessor();
+    PropertyKey.Builder keyBuilder = dataAccessor.keyBuilder();
+    String instanceName = helixManager.getInstanceName();
+    /*
```

Comment:
I suppose you are going to remove the commented code, right? It is here just for review. Please add a TODO or some clear indication that these lines will be removed before you merge

Reply:
Removed